cas-server-support-ldap/src/main/java/org/apereo/cas/authentication/PooledCompareCustomAttributeAuthenticationHandler.java

```diff
@@ -0,0 +1,72 @@
+package org.apereo.cas.authentication;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+import org.ldaptive.CompareOperation;
+import org.ldaptive.CompareRequest;
+import org.ldaptive.Connection;
+import org.ldaptive.LdapAttribute;
+import org.ldaptive.LdapException;
+import org.ldaptive.LdapUtils;
+import org.ldaptive.Response;
+import org.ldaptive.auth.AuthenticationCriteria;
+import org.ldaptive.auth.AuthenticationHandlerResponse;
+import org.ldaptive.auth.PooledCompareAuthenticationHandler;
+import org.ldaptive.pool.PooledConnectionFactory;
+
+/**
+ * Extended ldaptive {@code Authenticator} component, which uses a custom ldap
+ * attribute for password comparison.
+ *
+ * @author Robert Kornmesser
+ * @since 5.0.0
+ */
+public class PooledCompareCustomAttributeAuthenticationHandler extends PooledCompareAuthenticationHandler {
```

Comment:
I looked at 1.2 and I don't see it. I suggest you discuss this with Daniel and see if he'd be willing to take the contribution. If that works, we can simply just upgrade to 1.2 via that other PR, and accept your change afterwards. If not, we'll merge this PR as is.